### DISCUSSION TITLE
一文了解 Nebula Graph 上的 Spark 项目 - siwei.io

### DISCUSSION THREAD

**Discussion by giscus** [siwei-io/siwei-io.github.io#6] [Comments]
# 一文了解 Nebula Graph 上的 Spark 项目 - siwei.io

Nebula Graph 生态中有哪些 Spark 项目？ 本文为大家介绍 Spark-connector（包括 PySpark）， Nebula Algorithm 和 Nebula Exchange。

https://www.siwei.io/spark-on-nebula-graph/

**Comment by wey-gu** [comment]
If you would like to use another version of spark env, check their tags here: https://hub.docker.com/r/bde2020/spark-master/tags?page=1

and replace the image tag accordingly.
